group/var_ed25519/param.go

```diff
@@ -10,7 +10,7 @@
 // are isomorphic to curves having c == 1.
 //
 // For details see Bernstein et al, "Twisted Edwards Curves", http://eprint.iacr.org/2008/013.pdf
-package curve25519
+package var_ed25519
```

Comment:
Need to change package comment on line 1 here.

Reply:
Also renaming below "`func Param25519() *Param`" to "`func ParamEd25519() *Param`" might be best.